daemon/start.go

```diff
@@ -100,6 +100,8 @@ func (daemon *Daemon) containerStart(container *container.Container) (err error)
 	// setup has been cleaned up properly
 	defer func() {
 		if err != nil {
+			// start event is logged even on error
+			daemon.LogContainerEvent(container, "start")
```

Comment:
Comparing to `v1.10` code I don't think this is logged on mount errors etc. This should probably only be in the `containerd.Create` error block.